workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -424,14 +424,26 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
           padding: '0 1.5rem',
         },
         vertical: {
+          '& > div > div > button[class*="v5"]': {
+            alignItems: 'baseline',
+            textAlign: 'left',
+          },
+          '& > div > div > button > span': {
+            alignItems: 'baseline',
+            textAlign: 'left',
+          },
           borderBottom: `none`,
+          borderLeft: `1px solid ${general.tabsBottomBorderColor}`,
           padding: 0,
         },
         flexContainerVertical: {
           '& > button:hover': {
-            boxShadow: `-3px 0 ${general.tabsBottomBorderColor} inset`,
+            boxShadow: `2px 0 ${general.tabsBottomBorderColor} inset`,
           },
         },
+        indicator: {
+          left: 0,
```

Comment:
Also if I don't see an issue with the current solution, its better if we could unset right here:

```suggestion
          left: 0,
          right: unset,
```